src/xenia/app/library/game_scanner.cc

```diff
@@ -53,10 +55,14 @@ std::vector<std::filesystem::path> GameScanner::FindGamesInPath(
 
       // Do not attempt to scan SVOD data files
       auto filename = GetFileName(current_path);
-      if (filename == "Data") continue;
+      if (filename.string()._Starts_with("Data")) {
```

Comment:
Is this going to cause issues when compiling with clang?

Reply:
It probably will. Sadly starts_with and end_with is fully supported since c++20
We can use regex to check it instead

Reply:
probably better to roll out our own impl in xenia-base